### ISSUE TITLE
Do not report RCS1202 for extension methods

### ISSUE THREAD

**Issue by JosefPihrt** [JosefPihrt/Roslynator#790]
```csharp
using System.Collections.Generic;
using System.Linq;

class C
{
    void M()
    {
        var items = new List<C>();

        var s = items.LastOrDefault().GetString(); // RCS1202 should not be reported here
    }
}

static class E
{
    public static string GetString(this C c)
    {
        return c?.ToString() ?? "";
    }
}
```
